README.md

```diff
@@ -1 +1,34 @@
-# hw01
\ No newline at end of file
+---
+output:
+  pdf_document: default
+  word_document: default
+  html_document: default
+---
+# Caleb Weis
+## Computing for the Social Sciences
+### Homework 1
```

Comment:
Space out these titles. Every time you have a new header, image, list, etc. leave line breaks to prevent rendering issues